docs/integrations/github-actions.md

```diff
@@ -1,33 +1,107 @@
-# GitHub Actions
+# Cirrus Runners for GitHub Actions
 
-Tart already powers several CI services mentioned above including our own [Cirrus CI](https://cirrus-ci.org/guide/macOS/) which offers unlimited concurrency with per-second billing.
-For services that haven't leveraged Tart yet, we offer fully managed runners via a monthly subscription.
 *Cirrus Runners* is the fastest way to get your current CI workflows to benefit from Apple Silicon hardware. No need to manage infrastructure or migrate to another CI provider.
+Your actions will be executed in clean virtual machines with 4 M2 cores comparing to GitHub's own macOS Intel runners with just 3 cores.
```

Comment:
What do you think about clarify M2 (for those who don't follow Apple Silicon developments closely enough and might not know what M2 is) and emphasize the architecture differences:

```suggestion
Your actions will be executed in clean macOS virtual machines with 4 Apple M2 cores, compared to GitHub's own macOS runners with just 3 cores and only supporting the outdated Apple–Intel architecture.
```